tests/pytorch/mock_sparse/test_sddmm.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import pytest
+import dgl
+import dgl.backend as F
+import torch
+import numpy
+from dgl.mock_sparse import SparseMatrix
+parametrize_idtype = pytest.mark.parametrize("idtype", [F.int32, F.int64])
+parametrize_dtype = pytest.mark.parametrize('dtype', [F.float32, F.float64])
```

Comment:
Let's be consistent when we use single ' and double ".
Black will fix this for you.